### ISSUE TITLE
Proposed change to VisualC project settings

### ISSUE THREAD

**Issue by AntTheAlchemist** [libsdl-org/SDL#4328]
May I propose some small changes to the Visual Studio settings?

Runtime Library shouldn’t be Multi-threaded DLL (/MD). This will add a dependency for MSVC DLLs, which shouldn’t be needed by SDL’s core. Can this be changed to Multi-threaded (/MT)?

Should the default setting for enhanced instruction set be /arch:SSE? Shouldn’t it be /arch:IA32 for compatibility?

**Comment by slouken** [comment]
We use /NODEFAULTLIB to avoid a dependency on MSVC runtime. Have you actually seen a dependency there?

Regarding SSE flags, that's probably correct, though it hasn't been a problem in practice.

**Comment by AntTheAlchemist** [comment]
I used to get asked for MSVC DLLs on execution with default Visual Studio project settings. /MT seemed to fix it, I don't think it should be set to /MD for the same reason SDL doesn't depend on any DLLs.

This might have coincidentally stopped being an issue after some standard library calls inside SDL were changed to SDL's versions, which has happened a few times recently. It might be purely independent now.

Very rare for the SSE flags to be an issue unless someone's messing around on really old PCs I guess. It does set a minimum requirement for the published binaries that aren't described.

Both of these are pedantic issues and just minor suggestions :-)
